test/interface/test_filters.py

```diff
@@ -69,6 +74,15 @@ def multiple_quxs_inserted(session):
     session.commit()
 
 
+@pytest.fixture
+def multiple_corges_inserted(session, is_postgresql):
+    if is_postgresql:
+        corge_1 = Corge(id=1, name='name_1', tags=['foo'])
+        corge_2 = Corge(id=2, name='name_2', tags=['bar', 'baz'])
```

Comment:
It could be useful to add a couple of rows like the following:

```python
corge_3 = Corge(id=3, name='name_3', tags=['bar', 'foo', 'baz'])
corge_4 = Corge(id=4, name='name_4', tags=[])
```

This way we cover all the cases: we have a `corge` where all tags are `foo`, one where some tags are `foo`, one where none of the tags are `foo` and one with no tag values.

By doing that, it'll be reflected in the tests what happens when only some of the values are `foo` and how we expect `any` and `not_all` to behave.

Reply:
fixed with b0e1b4f47c2e746a92229b5e034a98230a4a14e4